result/bgpv4.yaml

```diff
@@ -24,35 +24,89 @@ components:
           description: >-
             The list of column names that the returned result set will contain.
             If the list is empty then all columns will be returned except for
-            any result_groups.
-            The name of the BGPv4 device cannot be excluded.
+            any result_groups. The name of the BGPv4 peer cannot be excluded.
           type: array
           items:
             type: string
             enum:
               - session_state
-              - routes_advertised
-              - routes_withdrawn
+              - flap_count
```

Comment:
in future, will we ever have something to count route flaps (instead of session flaps) ?

If so, I think calling this `session_flaps` makes more sense.

Reply:
No plan to add route flap count for now. We will only count session flaps and "session_flap_count" is more self explantory name.